azavea/climate-change-lab
# Feature/angular rc6

## Overview

Updates the Climate Lab app to Angular RC6. Being on older versions limits our ability to use up to date versions of third party libraries, and causes mismatches between our code and samples/howtos in the Angular official documentation.

### Notes

- ng2-bootstrap updated to v1.1.1
- Pulled in changes to ng2-auto-complete plugin, equivalent to v0.5.5, and applied an additional bugfix for RC6
- Bugfixed a change in the Indicator API endpoint transformer due to azavea/climate-change-api#91
- Bugfixed an issue with our auto-complete customization where the valueFormatter would only apply if the dropdown was open
- Moved auto-complete module to folder name matching the node package name for consistency

## Testing Instructions

* A clean install of node modules is best, so `rm -rf node_modules && npm install` before bringing up the dev server with `npm start`
* App should continue to function as before

Closes #35, #36
